po/pt.po

```diff
@@ -0,0 +1,476 @@
+# Translation of solaar's ui's messages to European Portuguese
+# Copyright (C) 2014 THE solaar'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the solaar package.
+#
+# Américo Monteiro <a_monteiro@gmx.com>, 2014.
+msgid ""
+msgstr ""
+"Project-Id-Version: solaar 0.9.2\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-07-23 22:38+0200\n"
+"PO-Revision-Date: 2014-09-14 20:57+0100\n"
+"Last-Translator: Américo Monteiro <a_monteiro@gmx.com>\n"
+"Language-Team: Portuguese <traduz@debianpt.org>\n"
+"Language: pt\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Lokalize 1.4\n"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "critical"
+msgstr "crítico"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "empty"
+msgstr "vazio"
+
+#: lib/logitech_receiver/i18n.py:38 lib/logitech_receiver/i18n.py:41
+msgid "full"
+msgstr "cheio"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "good"
+msgstr "bom"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "low"
+msgstr "baixo"
+
+#: lib/logitech_receiver/i18n.py:41
+msgid "almost full"
+msgstr "quase cheiro"
+
+#: lib/logitech_receiver/i18n.py:41
+msgid "discharging"
+msgstr "a descarregar"
+
+#: lib/logitech_receiver/i18n.py:41
+msgid "recharging"
+msgstr "a carregar"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "invalid battery"
+msgstr "bateria inválida"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "slow recharge"
+msgstr "carga lenta"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "thermal error"
+msgstr "erro térmico"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "device not supported"
+msgstr "dispositivo não suportado"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "device timeout"
+msgstr "tempo limite do dispositivo"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "sequence timeout"
+msgstr "tempo limite da sequência"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "too many devices"
+msgstr "demasiados dispositivos"
+
+#: lib/logitech_receiver/i18n.py:48
+msgid "Bootloader"
+msgstr "Gestor de arranque"
+
+#: lib/logitech_receiver/i18n.py:48 lib/solaar/ui/window.py:538
+msgid "Firmware"
+msgstr "Firmware"
+
+#: lib/logitech_receiver/i18n.py:48
+msgid "Hardware"
+msgstr "Hardware"
+
+#: lib/logitech_receiver/i18n.py:48
+msgid "Other"
+msgstr "Outro"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "closed"
+msgstr "fechado"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "open"
+msgstr "aberto"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "pairing lock is "
+msgstr "tranca de emparelhamento é "
```

Comment:
`o emparelhamente está `

`tranca` is not commonly used in this way in European Portuguese.